internal/worker/worker.go

```diff
@@ -187,6 +191,13 @@ func (worker *Worker) Run(ctx context.Context) error {
 			}
 		}
 
+		// Try to start the standby instance if no tasks are running
+		if worker.standby != nil && worker.tasks.Size() == 0 {
```

Comment:
I thought we've agreed to ditch resources at least for now in todays meeting 🤷 

What do you think about implementing them in a separate PR, since this is an optimization step?

Reply:
OK. But I just though that `TryStart` can check `tart-vms` label directly. No need to do all the resource manipulations like before.